snowflake/snowflake_test.go

```diff
@@ -102,6 +119,84 @@ func Test_SetNodeIDProvider_Ok(t *testing.T) {
 	assert.Equal(nodeID, decomposed.NodeID)
 }
 
+func Test_CurrentTimestamp_Ok(t *testing.T) {
+	assert := assert.New(t)
+	g := &Generator{
+		Settings: Settings{
+			Epoch:            time.Now().Add(-24 * time.Hour),
+			SequenceResolver: AtomicResolver,
+		},
+	}
+	v, e := g.currentTimestamp()
+	assert.GreaterOrEqual(v, int64(0))
+	assert.NoError(e)
+}
+
+func Test_CurrentTimestamp_EpochInTheFuture(t *testing.T) {
```

Comment:
@mysqto Looks like you swap the test case names between the epoch in the future & exceeds max time 😂